index.js

```diff
@@ -197,6 +197,28 @@ async function checkHeartbeat() {
   }
 }
 
+function convertStateArrayToObject(stateTransition) {
+  const statesTable = ['idle', 'initial_timer', 'duration_timer', 'stillness_timer']
+  const reasonsTable = ['movement', 'no_movement', 'door_open', 'initial_timer', 'duration_alert', 'stillness_alert']
+  const stateObject = {
+    state: statesTable[stateTransition[0]],
+    reason: reasonsTable[stateTransition[1]],
+    time: stateTransition[2],
+  }
+  return stateObject
+}
+
+// Sends a low battery alert if the time since the last alert is null or greater than the timeout
+async function sendLowBatteryAlert(location) {
+  const currentTime = await db.getCurrentTime()
+  const timeoutInMillis = helpers.getEnvVar('LOW_BATTERY_ALERT_TIMEOUT') * 1000
+  if (location.sentLowBatteryAlertAt === null || currentTime - location.sentLowBatteryAlertAt >= timeoutInMillis) {
```

Comment:
I've isolated the bug to this code block here. The left argument is working fine: if null, the alert is sent and the value is updated. The right argument is not working: timeout should be 1 day, but if `sentLowBatteryAlertAt` is before yesterday, no alert is sent.

Reply:
Hmmm, they must not be the values that you expect. I suggest a `console.log` to see what's in `currentTime`, `location.sentLowBatteryAlertAt`, and `timeoutInMillis`.

Reply:
ok! will the console.log messages show on the kubectl logs?

Reply:
Yes, they will :smile: